sanity-mz-2022/schemas/blog.js

```diff
@@ -0,0 +1,50 @@
+export default {
+  name: 'blog',
+  title: 'Blog',
+  type: 'document',
+  fields: [
+    {
+      name: 'image',
+      title: 'Image',
+      type: 'array',
+      of: [{ type: 'image'}],
+      options: {
+        hotspot: true
+      }
+    },
+    {
+      name: 'blogTitle',
+      title: 'BlogTitle',
```

Comment:
This is the title that appears in the Sanity Studio dashboard, so I'd try and make them more... "normal writing", as in Blog Title instead of BlogTitle
this applies to all the below ones as well